### ISSUE TITLE
Debug Failure. Unhandled SyntaxKind: EmptyStatement.

### ISSUE THREAD

**Issue by nalmabrcom** [annaet/cypress-tags#58]
Seems with the new version I've got this error. When I delete the `cypress-tags` from the project I don't see it. 
Here is my `plugins/index.js` file:
https://github.com/eclipse/che-che4z-lsp-for-cobol/blob/1fa56be609ac8371b69abba267988d8d5c57956f/tests/cypress/plugins/index.js#L50

Output: 
```
2021-05-28T06:25:59.511Z cypress:browserify get: C:\WORK\split_projects\che-che4z-lsp-for-cobol\tests\cypress\support\index.ts
2021-05-28T06:25:59.511Z cypress:browserify input: C:\WORK\split_projects\che-che4z-lsp-for-cobol\tests\cypress\support\index.ts
2021-05-28T06:25:59.511Z cypress:browserify output: C:\Users\mn892400\AppData\Roaming\Cypress\cy\production\projects\tests-fc1e24ed67cba6b9c22ad2b7ad9d039e\bundles\cypress\support\index.ts
2021-05-28T06:25:59.513Z cypress:browserify browserifyOptions: { extensions: [ '.ts', '.ts', '.tsx', '.ts', '.tsx' ], transform: [ [ 'C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\coffeeify\\index.js', [Object] ], [Function (anonymous)], [ 'C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\@cypress\\browserify-preprocessor\\lib\\simple_tsify', [Object] ] ], plugin: [], cache: {}, packageCache: {}, debug: true, entries: [ 'C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\cypress\\support\\index.ts' ] }
2021-05-28T06:25:59.516Z cypress:browserify making bundle C:\Users\mn892400\AppData\Roaming\Cypress\cy\production\projects\tests-fc1e24ed67cba6b9c22ad2b7ad9d039e\bundles\cypress\support\index.ts
2021-05-28T06:25:59.580Z cypress:server:plugins:child uncaught exception: {
  name: 'Error',
  message: 'Debug Failure. Unhandled SyntaxKind: EmptyStatement.',
  stack: 'Error: Debug Failure. Unhandled SyntaxKind: EmptyStatement.\n' +
    '    at pipelineEmitWithHintWorker (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104822:22)\n' +
    '    at pipelineEmitWithHint (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104372:17)\n' +
    '    at pipelineEmitWithComments (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:107816:13)\n' +
    '    at pipelineEmit (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104312:13)\n' +
    '    at emitExpression (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104296:13)\n' +
    '    at emitExpressionStatement (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:105720:13)\n' +
    '    at pipelineEmitWithHintWorker (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104507:32)\n' +
    '    at pipelineEmitWithHint (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104372:17)\n' +
    '    at pipelineEmitWithComments (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:107816:13)\n' +
    '    at pipelineEmit (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104312:13)\n' +
    '    at emit (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104285:13)\n' +
    '    at emitNodeList (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:107073:25)\n' +
    '    at emitList (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:106960:13)\n' +
    '    at emitBlockFunctionBodyWorker (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:106024:17)\n' +
    '    at emitBodyWithDetachedComments (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:107932:17)\n' +
    '    at emitBlockFunctionBody (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:106001:17)\n' +
    '    at emitSignatureAndBody (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:105940:21)\n' +
    '    at emitArrowFunction (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:105482:13)\n' +
    '    at pipelineEmitWithHintWorker (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104759:32)\n' +
    '    at pipelineEmitWithHint (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104372:17)\n' +
    '    at pipelineEmitWithComments (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:107816:13)\n' +
    '    at pipelineEmit (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104312:13)\n' +
    '    at emitExpression (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104296:13)\n' +
    '    at emitNodeList (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:107073:25)\n' +
    '    at emitExpressionList (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:106963:13)\n' +
    '    at emitCallExpression (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:105446:13)\n' +
    '    at pipelineEmitWithHintWorker (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104747:32)\n' +
    '    at pipelineEmitWithHint (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104372:17)\n' +
    '    at pipelineEmitWithComments (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:107816:13)\n' +
    '    at pipelineEmit (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104312:13)\n' +
    '    at emitExpression (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104296:13)\n' +
    '    at emitExpressionStatement (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:105720:13)\n' +
    '    at pipelineEmitWithHintWorker (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104507:32)\n' +
    '    at pipelineEmitWithHint (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104372:17)\n' +
    '    at pipelineEmitWithComments (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:107816:13)\n' +
    '    at pipelineEmit (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104312:13)\n' +
    '    at emit (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104285:13)\n' +
    '    at emitNodeList (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:107073:25)\n' +
    '    at emitList (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:106960:13)\n' +
    '    at emitBlockFunctionBodyWorker (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:106024:17)\n' +
    '    at emitBodyWithDetachedComments (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:107932:17)\n' +
    '    at emitBlockFunctionBody (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:106001:17)\n' +
    '    at emitSignatureAndBody (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:105940:21)\n' +
    '    at emitArrowFunction (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:105482:13)\n' +
    '    at pipelineEmitWithHintWorker (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104759:32)\n' +
    '    at pipelineEmitWithHint (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104372:17)\n' +
    '    at pipelineEmitWithComments (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:107816:13)\n' +
    '    at pipelineEmit (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104312:13)\n' +
    '    at emitExpression (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104296:13)\n' +
    '    at emitNodeList (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:107073:25)\n' +
    '    at emitExpressionList (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:106963:13)\n' +
    '    at emitCallExpression (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:105446:13)\n' +
    '    at pipelineEmitWithHintWorker (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104747:32)\n' +
    '    at pipelineEmitWithHint (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104372:17)\n' +
    '    at pipelineEmitWithComments (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:107816:13)\n' +
    '    at pipelineEmit (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104312:13)\n' +
    '    at emitExpression (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104296:13)\n' +
    '    at emitExpressionStatement (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:105720:13)\n' +
    '    at pipelineEmitWithHintWorker (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104507:32)\n' +
    '    at pipelineEmitWithHint (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104372:17)\n' +
    '    at pipelineEmitWithComments (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:107816:13)\n' +
    '    at pipelineEmit (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104312:13)\n' +
    '    at emit (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104285:13)\n' +
    '    at emitNodeList (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:107073:25)\n' +
    '    at emitList (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:106960:13)\n' +
    '    at emitSourceFileWorker (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:106691:13)\n' +
    '    at emitBodyWithDetachedComments (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:107932:17)\n' +
    '    at emitSourceFile (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:106612:21)\n' +
    '    at pipelineEmitWithHintWorker (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104380:24)\n' +
    '    at pipelineEmitWithHint (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104372:17)\n' +
    '    at pipelineEmit (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104312:13)\n' +
    '    at print (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104248:13)\n' +
    '    at writeFile (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104232:13)\n' +
    '    at Object.printFile (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\typescript\\lib\\typescript.js:104093:13)\n' +
    '    at processFile (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\cypress-tags\\dist\\index.js:176:28)\n' +
    '    at Stream.onend (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\cypress-tags\\dist\\index.js:185:20)\n' +
    '    at _end (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\through\\index.js:65:9)\n' +
    '    at Stream.stream.end (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\through\\index.js:74:5)\n' +
    '    at DestroyableTransform.onend (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\readable-stream\\lib\\_stream_readable.js:577:10)\n' +
    '    at Object.onceWrapper (events.js:421:28)\n' +
    '    at DestroyableTransform.emit (events.js:327:22)\n' +
    '    at endReadableNT (C:\\WORK\\split_projects\\che-che4z-lsp-for-cobol\\tests\\node_modules\\readable-stream\\lib\\_stream_readable.js:1010:12)\n' +
    '    at processTicksAndRejections (internal/process/task_queues.js:80:21)\n'
    ```

**Comment by nalmabrcom** [comment]
@annaet I think I found what causes the problem. 
`CYPRESS_EXCLUDE_TAGS`. Can you reproduce this issue?

**Comment by rodribera32** [comment]
@annaet I'm getting this error too. Do you know what could be the problem?
Using win 10, node v14, cypress @6.9.1 and @7.4.0

**Comment by omjokine** [comment]
Upgraded typescript from @4.2.4 to @4.3.2 and now I'm getting this error. So probably there's some mismatch between latest TypeScript and this plugin.

**Comment by annaet** [comment]
Sorry for the delayed response on this - thanks for reporting. I'll look into it and hopefully get a fix out soon.

**Comment by annaet** [comment]
@nalmabrcom 

I wasn't able to recreate this locally in my projects. Do you have a simple project where you can recreate this bug? I had a look at the repo you linked but I'm not sure how to run it.

**Comment by omjokine** [comment]
@annaet https://github.com/annaet/cypress-tags/runs/2873166875

**Comment by omjokine** [comment]
I was able to debug this issue and seems that `factory.createEmptyStatement()` causes this failure. Replaced it with `createOmittedExpression` which seems to work okay. However, I'm not 100% sure is there some side-effects with this, but probably not, because tests are running okay.
 
https://github.com/annaet/cypress-tags/pull/76

**Comment by annaet** [comment]
Thanks for the pull request @omjokine! I've merged your fix and published v0.2.0. This is working with the latest typescript on my projects, so hopefully this should fix the problem for you too.
